packages/types/ical.d.ts

```diff
@@ -168,6 +168,7 @@ declare module "ical.js" {
     public isNegative: boolean;
     public icalclass: string;
     public icaltype: string;
+    public toSeconds(): number;
```

Comment:
I did consider using `number | string` or even `any` type here to be accurate, but the rest of the ambient def doesn't do this so I chose not to.